gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.5-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.8-bin.zip
```

Comment:
if you changed the version here, why did the `gradle-wrapper.jar` not change? Note you can update the wrapper using this command:
```bash
./gradlew wrapper --gradle-version 8.8 --distribution-type all
```

Reply:
Not sure why it didn't update everything when I attempted the first time. From what I could tell when I initially updated things it had appeared to work because I was getting an error saying that 8.5 wasn't compatible and then fixed it when I ran my initial update attempt. I ran the provided command and that appears to have updated the files properly this time. Just pushed the updated files.

Reply:
thanks, but sorry, looks like previously we used the `bin` distribution type and we should keep it that way, so the correct command would be
```bash
./gradlew wrapper --gradle-version 8.8 --distribution-type bin
```
Could you run that again?